### PR TITLE
Fix invisible_recaptcha_tags to work in testing modes

### PR DESCRIPTION
**Pull request by rickychilcott**
Add relevant javascript and markup options to invisible_recaptcha_tags ONLY we are not in a testing mode

### CODE REVIEW

**Comment by rickychilcott** [comment]
I ran into a bug when I tried to run (previously passing) tests after adding the `recaptcha` gem and specifically using the `invisible_recaptcha_tags` method. This PR cleans up the markup so that it only passes through the needed details to get the UI to look the same, but not load the non-test related javascript.

Thanks for your work on this gem @ambethia!

**Comment by rickychilcott** [comment]
Also, let me know if you want a test for this. I just wrap the one section with an if clause, but I'm willing to prove it works.

**Comment by grosser** [comment]
looks reasonable to me ... a test would be nice though ...

File: lib/recaptcha/client_helper.rb [hunk 1]
```diff
@@ -57,24 +57,27 @@ def invisible_recaptcha_tags(options = {})
...
-      }
+      attributes = {}
+      fallback_uri = ""
 
-      hl = options.delete(:hl)
-      script_url = Recaptcha.configuration.api_server_url
-      script_url += "?hl=#{hl}" unless hl.to_s == ""
-      html << %(<script src="#{script_url}" async defer></script>\n) unless options.delete(:script) == false
-      fallback_uri = "#{script_url.chomp('.js')}/fallback?k=#{site_key}"
+      attributes["class"] = %(g-recaptcha #{options.delete(:class)})
```
Comment (grosser) [inline lib/recaptcha/client_helper.rb]: can just use quotes instead of `%(`

File: test/client_helper_test.rb [hunk 2]
```diff
@@ -77,5 +77,11 @@
       html = invisible_recaptcha_tags
       html.must_include(" data-sitekey=\"#{Recaptcha.configuration.site_key}\"")
     end
+
+    it "doesn't render script tag when in test mode" do
+      html = invisible_recaptcha_tags(env: "test")
+      html.wont_include("<script")
+      html.wont_include(" data-sitekey=\"#{Recaptcha.configuration.site_key}\"")
```
Comment (grosser) [inline test/client_helper_test.rb]: can make this more generic `"data-sitekey"`

File: test/client_helper_test.rb [hunk 3]
```diff
@@ -77,5 +77,11 @@
       html = invisible_recaptcha_tags
       html.must_include(" data-sitekey=\"#{Recaptcha.configuration.site_key}\"")
     end
+
+    it "doesn't render script tag when in test mode" do
```
Comment (grosser) [inline test/client_helper_test.rb]: more like "when verification is disabled" ?

**Comment by grosser** [comment]
sweet!

**Comment by rickychilcott** [comment]
Thanks for the review @grosser!

**Comment by grosser** [comment]
4.4.0 ... hope nobody comes and complains about their "usecase" :D

**Pull request merged**